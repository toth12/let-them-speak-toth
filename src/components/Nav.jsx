import React from 'react';
import { Link } from 'react-router-dom';
import img from '../assets/images/brand.svg';

const Nav = props => (
  <nav>
    <ul>
      <li><Link to={'/about'}>ABOUT</Link></li>
      <li><Link to={'/essays'}>ESSAYS</Link></li>
      <li><Link to={'/explore'}>EXPLORE</Link></li>
      <li className='brand'>
        <Link to={'/#'}><img src={img} /></Link>
      </li>
      <li><Link to={'/anthology'}>ANTHOLOGY</Link></li>
      <li><Link to={'/methods'}>METHODOLOGY</Link></li>
    </ul>
  </nav>
)

export default Nav;