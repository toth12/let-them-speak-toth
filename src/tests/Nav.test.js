import React from 'react';
import Nav from '../components/Nav';
import { createBrowserHistory } from 'history'
import { Router } from 'react-router';
import renderer from 'react-test-renderer';

it('renders', () => {
  const tree = renderer
    .create(
      <Router history={createBrowserHistory()}>
        <Nav/>
      </Router>
    ).toJSON();
  expect(tree).toMatchSnapshot();
});