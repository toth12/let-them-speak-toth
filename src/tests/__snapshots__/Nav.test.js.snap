// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<nav>
  <ul>
    <li>
      <a
        href="/about"
        onClick={[Function]}
      >
        ABOUT
      </a>
    </li>
    <li>
      <a
        href="/essays"
        onClick={[Function]}
      >
        ESSAYS
      </a>
    </li>
    <li>
      <a
        href="/explore"
        onClick={[Function]}
      >
        EXPLORE
      </a>
    </li>
    <li
      className="brand"
    >
      <a
        href="/#"
        onClick={[Function]}
      >
        <img
          src="test-file-stub"
        />
      </a>
    </li>
    <li>
      <a
        href="/anthology"
        onClick={[Function]}
      >
        ANTHOLOGY
      </a>
    </li>
    <li>
      <a
        href="/methods"
        onClick={[Function]}
      >
        METHODOLOGY
      </a>
    </li>
  </ul>
</nav>
`;
